src/integration-test/java/com/commercetools/sync/integration/commons/utils/SphereClientUtils.java

```diff
@@ -19,6 +21,18 @@ public class SphereClientUtils {
   public static final SphereClient CTP_SOURCE_CLIENT = createClient(CTP_SOURCE_CLIENT_CONFIG);
   public static final SphereClient CTP_TARGET_CLIENT = createClient(CTP_TARGET_CLIENT_CONFIG);
```

Comment:
So these clients won't be replaced by the v2?

Reply:
It won't be. At this moment we still keep old `SphereClient` to avoid breaking change. That's why our integration test still make use of old `SphereClient`.  Regarding the test of new `SphereClient`, I have discussed with @butenkor, we can simply add one integration test and benchmark test in ProductSync to test it.